src/js/components/TaskView.js

```diff
@@ -99,21 +112,36 @@ class TaskView extends mixin(SaveStateMixin, StoreMixin) {
     this.setState({checkedItems: {}, killAction: ''});
   }
 
-  filterByCurrentStatus(tasks) {
-    let status = this.state.filterByStatus;
-    if (status === 'all') {
+  filterByCurrentStatus() {
```

Comment:
We still need to pass in the tasks here! Maybe we should call them `filteredTasks` because we're passing in an already filtered set from L140